src/language-css/printer-postcss.js

```diff
@@ -289,15 +289,22 @@ function genericPrint(path, options, print) {
       return adjustStrings(node.value, options);
     }
     case "selector-tag": {
+      const parentNode = path.getParentNode();
+      const index = parentNode && parentNode.nodes.indexOf(node);
```

Comment:
I think you can get this with `path.getKey()`.

Reply:
@j-f1 I can't find `path.getKey()` in `fast-path.js` :confused:

Reply:
Oops, it’s `getName`. However, it always returns a string.

Reply:
@j-f1 we can't get name preview node without getting index current node, approve this PR?

Reply:
@evilebottnawi I didn’t quite understand your comment; can you clarify please?